### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/agile#17]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://pages.18f.gov/agile/
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #0069be.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `#repositories > ul > li:nth-child(2) > a > h4`
  - `<h4>agile</h4>`
- **Error** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #0069be.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - `#repositories > ul > li:nth-child(2) > a > p`
  - `<p>Main repository</p>`
### Summary

**Errors:** 2

**Comment by porta-antiporta** [comment]
Checking with #g-accessibility to confirm the latest audit I ran confirms these have been addressed.
One item that is also needed is an update to the logo https://github.com/18F/agile/pull/45.

I _suspect_ it doesn't show up in the scan because it's an image file.  Though the image itself (being a text based image) is the old logo (which wasn't compliant).

**Comment by porta-antiporta** [comment]
Note: the audit was run using the Accessibility Developer Tools in Chrome, not the HTML-Codesniffer used to originally submit the issue.

**Comment by suprenant** [comment]
@porta-antiporta since we've addressed the logo issue and moved to the new template, I think we can close this up. How does that sound to you?

**Comment by suprenant** [comment]
@porta-antiporta re-ping
